src/store/RootStore/QueryParamsStore/QueryParamsStore.ts

```diff
@@ -0,0 +1,74 @@
+import { action, makeObservable, observable } from 'mobx';
+import * as qs from 'qs';
+
+type PrivateFields = '_params' | '_count';
+
+export default class QueryParamsStore {
+  private _params: qs.ParsedQs = {};
+  private _search: string = '';
+  private _count: number = 0;
+  private _rootStore;
+
+  constructor(rootStore: any) {
```

Comment:
тоже лучше тип не отметать:
```suggestion
  constructor(rootStore: RootStore) {
```